tutorials/Tutorial_For_QA_Registration.ipynb

```diff
@@ -0,0 +1,377 @@
+    "from ndmg.utils.qa_utils import get_min_max, opaque_colorscale, pad_im\n",
+    "from argparse import ArgumentParser\n",
+    "from scipy import ndimage\n",
+    "from matplotlib.colors import LinearSegmentedColormap\n",
+    "from nilearn.plotting.edge_detect import _edge_map as edge_map\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Saving figures\n",
+    "Save the result returned by function plot_overlays and name the file as \"registered image \\_2\\_ reference image\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def reg_mri_pngs(\n",
+    "    mri, atlas, outdir, loc=0, mean=False, minthr=2, maxthr=95, edge=False\n",
+    "):\n",
+    "    \"\"\"\n",
+    "    A function to create and save registered brain slice figures.\n",
+    "    \n",
+    "    Parameter\n",
+    "    ---------\n",
+    "    mri: nifti file\n",
+    "        the registered brain file generated in each registration step.\n",
+    "    atlas: nifti file\n",
+    "        the reference brain file used in each registration step.\n",
+    "    outdir: str\n",
+    "        directory where output png file is saved. \n",
+    "    loc: int\n",
+    "        which dimension of the 4d brain data to use \n",
+    "    mean: bool\n",
+    "        whether to calculate the mean of the 4d brain data \n",
+    "        If False, the loc=0 dimension of the data (mri_data[:, :, :, loc]) is used \n",
+    "    minthr: int\n",
+    "         lower percentile threshold  \n",
+    "    maxthr: int\n",
+    "         upper percentile threshold \n",
+    "    \"\"\"\n",
+    "    atlas_data = nb.load(atlas).get_data()\n",
+    "    mri_data = nb.load(mri).get_data()\n",
+    "    if mri_data.ndim == 4:  # 4d data, so we need to reduce a dimension\n",
+    "        if mean:\n",
+    "            mr_data = mri_data.mean(axis=3)\n",
+    "        else:\n",
+    "            mr_data = mri_data[:, :, :, loc]\n",
+    "    else:  # dim=3\n",
+    "        mr_data = mri_data\n",
+    "\n",
+    "    cmap1 = LinearSegmentedColormap.from_list(\"mycmap1\", [\"white\", \"magenta\"])\n",
+    "    cmap2 = LinearSegmentedColormap.from_list(\"mycmap2\", [\"white\", \"green\"])\n",
+    "\n",
+    "    fig = plot_overlays(atlas_data, mr_data, [cmap1, cmap2], minthr, maxthr, edge)\n",
+    "    # name and save the file\n",
+    "    fig.savefig(outdir + \"/\" + get_filename(mri) + \"_2_\" + get_filename(atlas) + \".png\", format=\"png\") \n",
+    "    \n",
+    "    #plt.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generating the qa overlay figures\n",
+    "First overlap the registered image and the reference image, then select the appropriate position to cut nine slices and return the figure foverlay."
```

Comment:
"... then select the appropriate position to cut nine slices (3 slices each from sagittal, axial, and coronal) and return..."

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/7b32b3d114ee8d7a65060fb0dc1fefa8393cb052